evap/results/templates/results_course_detail.html

```diff
@@ -103,19 +103,26 @@
                 <span class="panel-title">{% trans "Contributors" %}</span>
             </div>
             <div class="panel-body">
-                {% for contributor,sections in contributor_sections.items %}
-                    {% regroup sections by questionnaire as sections_grouped %}
+                {% for contributor, contributor_data in contributor_sections.items %}
```

Comment:
please add a `<div class="panel-group">` around the panels created in this for loop